classes/Product.php

```diff
@@ -4742,6 +4742,8 @@ public static function getProductProperties($id_lang, $row, Context $context = n
         if (isset($row['quantity_wanted'])) {
             // 'quantity_wanted' may very well be zero even if set
             $quantity = max((int) $row['minimal_quantity'], (int) $row['quantity_wanted']);
+        } elseif (isset($row['cart_quantity'])) {
+            $quantity = max((int) $row['minimal_quantity'], (int) $row['cart_quantity']);
         } else {
             $quantity = (int) $row['minimal_quantity'];
```

Comment:
Could it be simpler to have
```
$quantity = max(
  (int) $row['minimal_quantity'],
  isset($row['quantity_wanted']) ? (int) $row['quantity_wanted'] : 0,
  isset($row['cart_quantity']) ? (int) $row['cart_quantity'] : 0,
);
```

Reply:
Or is there an importance about order. quantity_wanted before cart_quantity ?

Reply:
I'm actually not sure, since `cart_quantity` was not managed I think leaving `quantity_approved` the priority reduces the risk of side effects bug
Although I don't think they are used in the same use cases it is safer